src/components/controls/VersionControl/VersionControl.js

```diff
@@ -0,0 +1,32 @@
+// Controlled selector that allows a user to switch between CMIP5 and
+// CMIP6 datasets. 
+
+import React from 'react';
+import Select from 'react-select';
+
+export default function VersionControl( { onSelect, selected } ) {
+	// this is all just hard-coded, it's complocated to
+	// get it from the API.
+	const items = [
+		{"value": "CMIP6", "label": "CMIP6+CWEC2020"},
+		{"value": "CMIP5", "label": "CMIP5+CWEC2016"}
+	];
+	
+	const label = "Future-shifted Dataset";
+	
+	const value = () => {
+		return items.find((o) => o["value"] === selected)
+	};
+	
+	return (
+		<div>
+		{label}
+		<Select
+		  isLoading={items.length === 0}
+		  options={items}
+		  value={value()}
+		  onChange={onSelect}
```

Comment:
We could sweeten (and simplify for users) the interface by returning `o.value`, just as the input value is mapped in the other direction. Then the outside world wouldn't need to know about the structure of the options list.